server/src/definition/ModelDefinitionProvider.ts

```diff
@@ -35,14 +35,15 @@ export class ModelDefinitionProvider implements DbtNodeDefinitionProvider {
 
       const isPackageSpecified = matches.length === 2;
       const wordAbsoluteOffset = document.offsetAt(getAbsolutePosition(jinja.range.start, wordRange.start));
-      let dbtPackage;
+      let dbtPackage = undefined;
       let model;
       let packageSelectionRange;
       let modelSelectionRange;
 
       if (isPackageSpecified) {
-        dbtPackage = matches[0].text;
-        model = matches[1].text;
+        dbtPackage = matches[0].text.slice(1, -1);
```

Comment:
Can we add static function and use it? Every time I'm trying to understand what is going here)
```
withoutQuotes(s) {
return s.slice(1, -1);
}
```

Reply:
Added new function `truncateAtBothSides`.